hr_emergency_contact/models/hr_employee.py

```diff
@@ -3,20 +3,17 @@
 # © 2016 OpenSynergy Indonesia
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models, fields
+from odoo import models, fields
 
 
 class HrEmployee(models.Model):
 
     _inherit = 'hr.employee'
 
     emergency_contact_ids = fields.Many2many(
-        string='Emergency Contacts',
         comodel_name='res.partner',
-        relation='rel_employee_emergency_contact',
```

Comment:
@andhit-r Hum, good question. Frankly I can't tell as I don't have enough experience on migrations. But I feel like it's an old fashioned way of describing the relation and we should let the ORM do the job.